internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,55 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ElenaMask/go1fl-sprint5-final/internal/personaldata"
+	"github.com/ElenaMask/go1fl-sprint5-final/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	s := strings.Split(datastring, ",")
+	if len(s) != 2 {
+		return fmt.Errorf("invalid argument datastring = %s", datastring)
+	}
+	steps, err := strconv.Atoi(s[0])
+	if err != nil {
+		return err
```

Comment:
Лучше все ошибки оборачивать, так быстрее можно будет понять, где она возникла.
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```